libraries/AP_HAL_SITL/Synth.hpp

```diff
@@ -101,7 +101,7 @@ double amplitude(double dTime, sEnvelope env)
     else if (dTime > env.dAttackTime && dTime <= (env.dAttackTime + env.dDecayTime))                                                       // Decay phase
         dAmplitude = ((dTime - env.dAttackTime) / env.dDecayTime) * (env.dSustainAmplitude - env.dStartAmplitude) + env.dStartAmplitude;
 
-    else if (dTime <= env.dAttackTime)                                                                                                     // Attack phase
+    else if (!iszero(env.dAttackTime) && dTime <= env.dAttackTime)                                                                                                     // Attack phase
```

Comment:
this cannot access to AP_MATH iszero function, just add
(env.dAttackTime < DBL_EPSILON) instead